src/com/magento/idea/magento2plugin/magento/files/commands/DeleteEntityByIdCommandFile.java

```diff
@@ -19,7 +19,7 @@
     public static final String WEB_API_METHOD_NAME = "execute";
     private static final String DIRECTORY = "Command";
     private static final String WEB_API_INTERFACE_NAME_PATTERN = "Delete%entityName%ByIdInterface";
-    private static final String WEB_API_URL_PATTERN = "%entityName%/delete-by-id";
+    private static final String WEB_API_URL_PATTERN = "%entityName%/delete-by-id/:entityId";
```

Comment:
Let's keep it simpler like `/V1/employee/:entityId`, since we specify the `DELETE` method, there is no reason have additional info in route.

Reply:
Thank you! I'll fix that in a moment!

Reply:
@eduard13, I have done that:
<img width="1026" alt="Screenshot 2021-06-24 at 16 54 20" src="https://user-images.githubusercontent.com/31848341/123275495-092a7280-d50d-11eb-9bec-f252c793dfca.png">